src/codeu/chat/util/Time.java

```diff
@@ -74,6 +74,10 @@ public static Time now() {
     return Time.fromMs(System.currentTimeMillis());
   }
 
+  public static Time minTime() {
```

Comment:
Sorry, didn't see this before merging. I'll just quickly fix it and push it directly.